docs/localizability.md

```diff
@@ -149,4 +151,5 @@ private void DoSomething(ILogger logger)
 [8]: https://github.com/NuGet/NuGet.Build.Localization/blob/931c5f21742fff61c7f53a19039b89dddc7a01cd/localize/comments/15/NuGet.Packaging.dll.lci#L58-L66
 [9]: https://github.com/dotnet/sign/blob/ef0e6b3ef8281dff1d62cea34445bd88fc3e6714/src/Sign.Core/Resources.resx#L131-L134
 [10]: https://aka.ms/allaboutloc
-[11]: https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/20709/MicroBuild-Localization
\ No newline at end of file
+[11]: https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/20709/MicroBuild-Localization
+[12]: https://github.com/NuGet/NuGet.Client/pull/6170
```

Comment:
Similar to another comment, it's not clear to me why someone reading these docs would be more interested in this pull request than any other pull request that modified `build/loc.proj` or the build yaml.

Reply:
This pull request contains an explanation of why it's important to sign the localized assemblies, as well as how it's being done. It's not accidental that they're signed, it is intentional, so the documentation is referencing that PR to point to the purpose of that behavior.